src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,27 @@
 package mate.jdbc;
 
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.dao.ManufacturerDao;
+import mate.jdbc.model.entity.Manufacturer;
+
 public class Main {
-    public static void main(String[] args) {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
 
+    public static void main(String[] args) {
+        ManufacturerDao manufacturerDao = (ManufacturerDao)
+                injector.getInstance(ManufacturerDao.class);
+        Manufacturer hxtm = new Manufacturer();
+        hxtm.setName("Bucephalus").setCountry("Ukraine");
+        Manufacturer rheinmetall = new Manufacturer();
+        rheinmetall.setName("Marder").setCountry("Germany");
```

Comment:
```suggestion
        Manufacturer hxtm = new Manufacturer();
        hxtm.setName("Bucephalus").setCountry("Ukraine");
        Manufacturer rheinmetall = new Manufacturer();
        rheinmetall.setName("Marder").setCountry("Germany");
```

hxtm, rheinmetall not informative variable names